tests/test_cli.py

```diff
@@ -292,17 +292,37 @@ def test_add_namespace(self, caplog, gateway):
 
     def test_add_namespace_ipv6(self, caplog, gateway):
         caplog.clear()
-        cli(["--server-address", server_addr_ipv6, "namespace", "add", "--subsystem", subsystem, "--rbd-pool", pool, "--rbd-image", image])
+        cli(["--server-address", server_addr_ipv6, "namespace", "add", "--subsystem", subsystem, "--rbd-pool", pool, "--rbd-image", image, "--force"])
         assert f"Adding namespace 3 to {subsystem}, load balancing group 1: Successful" in caplog.text
+        assert f'will continue as the "force" argument was used' in caplog.text
         caplog.clear()
         cli(["--format", "json", "namespace", "list", "--subsystem", subsystem, "--nsid", "3"])
         assert '"load_balancing_group": 0' in caplog.text
-        cli(["--server-address", server_addr_ipv6, "namespace", "add", "--subsystem", subsystem, "--nsid", "8", "--rbd-pool", pool, "--rbd-image", image])
+        cli(["--server-address", server_addr_ipv6, "namespace", "add", "--subsystem", subsystem, "--nsid", "8", "--rbd-pool", pool, "--rbd-image", image, "--force"])
         assert f"Adding namespace 8 to {subsystem}, load balancing group 1: Successful" in caplog.text
         caplog.clear()
         cli(["--format", "json", "namespace", "list", "--subsystem", subsystem, "--nsid", "8"])
         assert '"load_balancing_group": 0' in caplog.text
 
+    def test_add_namespace_same_image(self, caplog, gateway):
+        caplog.clear()
```

Comment:
suggesting adding comments to explain the purpose of the test, for each part (after every caplog.clear()..)

Reply:
@barakda we need to go over all the tests and add such comments. This would be done in a separate issue.